src/java/picard/illumina/IlluminaBasecallsToFastq.java

```diff
@@ -68,15 +68,32 @@
 import java.util.Set;
 
 @CommandLineProgramProperties(
-        usage = "Generate fastq file(s) from data in an Illumina basecalls output directory.\n" +
-                "Separate fastq file(s) are created for each template read, and for each barcode read, in the basecalls.\n" +
-                "Template fastqs have extensions like .<number>.fastq, where <number> is the number of the template read,\n" +
-                "starting with 1.  Barcode fastqs have extensions like .barcode_<number>.fastq, where <number> is the number\n" +
-                "of the barcode read, starting with 1.",
-        usageShort = "Generate fastq file(s) from data in an Illumina basecalls output directory",
-        programGroup = Illumina.class
+  usage = IlluminaBasecallsToFastq.USAGE_SUMMARY + IlluminaBasecallsToFastq.USAGE_DETAILS,
+  usageShort = IlluminaBasecallsToFastq.USAGE_SUMMARY,
+  programGroup = Illumina.class
 )
 public class IlluminaBasecallsToFastq extends CommandLineProgram {
+  static final String USAGE_SUMMARY = "Generate fastq file(s) from Illumina basecall read data.  ";
+  static final String USAGE_DETAILS = "<p>This tool generates fastq files from data in an Illumina basecalls (*.bcl) output directory.  " +
+          "Separate fastq files are created for each template, barcode, and index read.   In the absence of sample pooling (multiplexing) and/or barcodes, then an OUTPUT_PREFIX (file) must be provided as the sample identifier.  For multiplexed samples, a MULTIPLEX_PARAMS (file) must be used.  The MULTIPLEX_PARAMS file contains the list of sample barcodes used to sort template, barcode, and index reads.  It is essentially the same as the BARCODE_FILE used in the" +
```

Comment:
OK, the index read is indeed a thing. In a bit more detail for posterity: (http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3245947/)

> On the Illumina sequencing platform, the perhaps most widely used multiplexing strategy (the vendor's protocol) uses indexes, which are embedded within one of the adapters (8–10), separated from the actual template (Figure 1A). Thus, for a typical Illumina multiplex library, the index is sequenced after the forward read in a separate ‘index read’, for which a new sequencing primer is annealed. Although there are alternative indexing approaches, where the index is attached adjacent to the insert [e.g. (7)], this strategy has several benefits. Decoupling the actual template read and the index read allows the index read to be left out if not required and also keeps the sequencing error rate low, because phasing (3,11,12), one of the main sources of sequencing error on the Illumina platform, is reset with the annealing of a new sequencing primer. In addition, image analysis and the estimation of base calling parameters are not affected by the frequently unbalanced base composition of indexes.